### PR TITLE
Create ui-c8y-1018-503-50-added-skip-to-passthrough-test-graft-release-y-2024-fix-dm-3348-added-skip-to-passthrough-test-graft-release-y-2024.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by dmaj

Ticket: [DM-3348](https://cumulocity.atlassian.net/browse/DM-3348)
Original PR: [5618](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5618)

### CODE REVIEW

File: content/change-logs/ui-c8y-1018-503-50-added-skip-to-passthrough-test-graft-release-y-2024-fix-dm-3348-added-skip-to-passthrough-test-graft-release-y-2024.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+date: ""
+title: added skip to passthrough test [GRAFT] [release/y2024]
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1018-503-50-added-skip-to-passthrough-test-graft-release-y-2024-fix-dm-3348-added-skip-to-passthrough-test-graft-release-y-2024.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Added option to skip passthrough test
```

File: content/change-logs/ui-c8y-1018-503-50-added-skip-to-passthrough-test-graft-release-y-2024-fix-dm-3348-added-skip-to-passthrough-test-graft-release-y-2024.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3348
+version: 1018.503.50
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1018-503-50-added-skip-to-passthrough-test-graft-release-y-2024-fix-dm-3348-added-skip-to-passthrough-test-graft-release-y-2024.md:17]: The Claude AI suggested to use the following body:
```suggestion
A new option has been added to skip the passthrough test if needed. This improves flexibility when running tests by allowing the passthrough test to be optionally skipped.

The change was made in response to ticket DM-3348. It has been merged into the y2024 release branch.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@janhommes It seems that this has been branched from develop and should be merged to 2024. This does not work.

**Comment by janhommes** [comment]
@BeateRixen yeah there is still an issue I will work on today. Closing this as anyhow not relevant as this is a test change.

**Pull request closed without merging**